packages/swapper/src/swappers/RelaySwapper/constant.ts

```diff
@@ -0,0 +1,57 @@
+import type { ChainId } from '@shapeshiftoss/caip'
+import {
+  arbitrumChainId,
+  avalancheChainId,
+  baseChainId,
+  btcChainId,
+  ethChainId,
+  gnosisChainId,
+  optimismChainId,
+  polygonChainId,
+  solanaChainId,
+} from '@shapeshiftoss/caip'
+import { KnownChainIds } from '@shapeshiftoss/types'
+import { zeroAddress } from 'viem'
+import {
+  arbitrum,
+  avalanche,
+  base,
+  gnosis,
+  mainnet as ethereum,
+  optimism,
+  polygon,
+} from 'viem/chains'
+
+import type { SupportedChainIds } from '../../types'
+
+export const relaySupportedChainIds = [
+  KnownChainIds.EthereumMainnet,
+  KnownChainIds.ArbitrumMainnet,
+  KnownChainIds.BaseMainnet,
+  KnownChainIds.OptimismMainnet,
+  KnownChainIds.PolygonMainnet,
+  KnownChainIds.GnosisMainnet,
+  KnownChainIds.AvalancheMainnet,
+  KnownChainIds.BitcoinMainnet,
+  KnownChainIds.SolanaMainnet,
+]
+
+export const relayChainMap: Record<ChainId, number> = {
+  [btcChainId]: 8253038,
```

Comment:
Magic numbers, comment me out with link to docs

Reply:
Done in [29fc44d](https://github.com/shapeshift/web/pull/9188/commits/29fc44d8ce158086f2cbec2c3c7ee0dd44302dac)